sales/amazon_connector/features.rst

```diff
@@ -0,0 +1,37 @@
+=======================================
+Synchronization between Amazon and Odoo
+=======================================
+
+The **Amazon Connector** synchronizes the orders between Amazon and your Odoo database, which
+reduces considerably the amount of time spent on your Amazon Marketplace dashboard, making your
+daily routine a lot easier. More advanced tasks still require you to connect to your Amazon account.
+
+This connector is able to:
+
++--------------------------------------------------+----------------------------------------+
+| From Amazon to Odoo                              | From Odoo to Amazon                    |
++==================================================+========================================+
+| Synchronize all confirmed orders (both FBA and   | Notify Amazon of a confirmed shipping  |
+| FBM) and their order items which include:        | (FBM) in order to get paid.            |
+|                                                  |                                        |
+| - the product's details (SKU, listing name &     |                                        |
```

Comment:
Let's drop the mention of product details. This is implicit and not even exhaustive anyway.

Reply:
I removec the details between brackets but I think it's still informative :)